src/Controller/RequestsController.php

```diff
@@ -34,12 +31,18 @@ class RequestsController extends Controller
      */
     public function beforeFilter(Event $event)
     {
-        // TODO add config override
-        if (!Configure::read('debug')) {
-            throw new NotFoundException();
-        }
-
         $this->response->header(['Content-Security-Policy' => '']);
+
+        parent::beforeFilter($event);
+    }
+
+    /**
+     * Do not load components for this controller.
+     *
+     * @return void
+     */
+    public function initialize()
```

Comment:
I thought it was, but that one controller has issues with the request handler. There are tests that say it's a JSON response but it has an HTML only view. Either I dig deep into the code to see why or revert the behavior for this one case.